.travis.yml

```diff
@@ -1,38 +1,25 @@
 sudo: false
 language: python
 dist: xenial # default is ubuntu 14.04, we need at least 16.04
-python: # test various python versions (Django Support for various python versions is described within the exclude: section)
-  - "2.7"
-  - "3.4"
+python: # test various python versions
   - "3.5"
   - "3.6"
   - "3.7"
-  - "pypy"  # PyPy2
-  - "pypy3" # PyPy3
+  - "3.8"
+  - "pypy3"
 env:
-  - DJANGO=1.11
-  - DJANGO=2.0
-  - DJANGO=2.1
   - DJANGO=2.2
+  - DJANGO=3.0
+  - DJANGO=3.1
   - DJANGO=master
 matrix:
   exclude:
-    # Django >= 2.0 does not support Python2.7/PyPy2 anymore
-    - { python: "2.7", env: DJANGO=2.0 }
-    - { python: "pypy", env: DJANGO=2.0 }
-    - { python: "2.7", env: DJANGO=2.1 }
-    - { python: "pypy", env: DJANGO=2.1 }
-    - { python: "2.7", env: DJANGO=2.2 }
-    - { python: "pypy", env: DJANGO=2.2 }
-    - { python: "2.7", env: DJANGO=master }
-    - { python: "pypy", env: DJANGO=master }
-    # Django >= 2.1 does not support Python3.4 anymore
-    - { python: "3.4", env: DJANGO=2.1 }
-    - { python: "3.4", env: DJANGO=2.2 }
-    - { python: "3.4", env: DJANGO=master }
+    # Django >= 3.0 does not support Python3.5 anymore
+    - { python: "3.5", env: DJANGO=3.0 }
+    - { python: "3.5", env: DJANGO=3.1 }
+    - { python: "3.5", env: DJANGO=master }
```

Comment:
FYI, according to https://docs.djangoproject.com/en/3.1/faq/install/#what-python-version-can-i-use-with-django Django 3.0+ does not support Python 3.5. Support starts with Python 3.6.

Reply:
Never mind, I just saw that this is the `exclude` part

:+1: